apigentools/commands/init.py

```diff
@@ -16,6 +16,52 @@
 
 
 class InitCommand(Command):
+    config_file_json = {
+                    "codegen_exec": "openapi-generator",
+                    "languages": {},
+                    "server_base_urls": {
+                        "v1": "https://api.myserver.com/v1",
+                        },
+                    "spec_sections": {
+                        "v1": [],
+                        },
+                    "spec_versions": ["v1"],
+    }
+    v1_header_json = {
```

Comment:
Since these are class level constants, lets uppercase the variable names - https://www.python.org/dev/peps/pep-0008/#constants

Reply:
I made them uppercase.